### ISSUE TITLE
NullReferenceException on Dashboard failed job

### ISSUE THREAD

**Issue by marcselman** [HangfireIO/Hangfire#1165]
I have several pages of failed jobs in my Hangfire dashboard but there's one page I can't view.
I get the following stacktrace:

```
[NullReferenceException: Object reference not set to an instance of an object.]
   Hangfire.Dashboard.Pages.FailedJobsPage.Execute() +1874
   Hangfire.Dashboard.RazorPage.TransformText(String body) +30
   Hangfire.Dashboard.RazorPageDispatcher.Dispatch(DashboardContext context) +85
   Hangfire.Dashboard.<>c__DisplayClass1_2.<UseHangfireDashboard>b__1(IDictionary`2 env) +465
```

Using v1.6.17

**Comment by AndyWorral** [comment]
Might be related to: [https://github.com/HangfireIO/Hangfire/issues/1164 ](https://github.com/HangfireIO/Hangfire/issues/1164 )

**Comment by pieceofsummer** [comment]
@AndyWorral not really. #1164 was a regression introduced in 1.6.18, and shouldn’t be the case for 1.6.17 and older versions. Stack trace is also completely different.

**Comment by marcselman** [comment]
@pieceofsummer That's right. Do you have any idea what this could be? Can I provide you with more information?

**Comment by pieceofsummer** [comment]
@marcselman no idea. It's kinda hard to debug this because offsets are JITted code offsets, which depend on your current framework and architecture and may be hard to reproduce. Can you provide a memory dump of your Dashboard process?
